.github/stale.yml

```diff
@@ -0,0 +1,22 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 30
+
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```

Comment:
Neither of these labels are in current use.
Perhaps `Confirmed Bug` would be a good starting label.

Reply:
Resolved, thanks!